src/control/build/interop.go

```diff
@@ -103,6 +114,25 @@ var defaultRules = []InteropRule{
 	},
 }
 
+var (
+	Server22xAgent20x = &InteropRule{
+		Self:          ComponentServer,
+		Other:         ComponentAgent,
+		Description:   "server v2.2.x is compatible with agent v2.0.x",
+		StopOnSuccess: true,
+		Check: func(self, other *VersionedComponent) bool {
+			// We assume that an unversioned agent is 2.0.x.
+			v0_0_0 := MustNewVersion("0.0.0")
+			v2_2_0 := MustNewVersion("2.1.100")
```

Comment:
what about e.g. 2.1.101 ? we have builds between 2.1.100-2.1.106

Reply:
`2.1.100` is treated as the base version for "2.2.x" in order to accommodate pre-release and release versions.